sparkstreaming/src/test/java/zipkin/sparkstreaming/StorageConsumerTest.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.sparkstreaming;
+
+import org.junit.Test;
+
+public class StorageConsumerTest {
+
+  @Test
+  public void test() {
+    // TODO
```

Comment:
I will write this.. currently tested manually both in dev mode and also to a spark cluster